modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -317,7 +317,7 @@
 			</argument>
 			<description>
 				The natural exponential function. It raises the mathematical constant [b]e[/b] to the power of [code]s[/code] and returns it.
-				[b]e[/b] has an approximate value of 2.71828.
+				[b]e[/b] has an approximate value of 2.71828, and can be obtained with [code]exp(1)[/code].
```

Comment:
Should be something like ", and is equivalent to [code]exp(1)[/code]". We don't want to seem as if we're encouraging actually writing `exp(1)`.

Reply:
Are there other ways to get the constant e in codot?

Reply:
Oh, never mind, I thought we had `e` as an exposed constant, but it's only exposed in C#. Keep this as-is.